### DISCUSSION TITLE
Using wavy sliders?

### DISCUSSION THREAD

**Discussion by mahozad** [zyrouge/symphony#431] [Ideas]
Hi,

I'm doing a self-promotion here :)
Please ignore it if you are not interested.

Would you like to use the [wavy-slider](https://github.com/mahozad/wavy-slider) in your app?
I think Android 13 uses this kind of slider for its media player.

[Demo](https://mahozad.ir/wavy-slider).

**Comment by zyrouge** [comment]
Maybe? But, I don't think it's very user friendly.

**Comment by Azrael2925** [comment]
Truly amazing app you've built, would you consider adding multi selection and tap and hold functionality anytime soon. Picking songs individually for a big playlist can be tiring 😅

**Reply by zyrouge** [reply]
Those are coming.

**Reply by Azrael2925** [reply]
Yo, wanted to also ask if stuff like individual playlist management [option to manage songs in playlists by pulling them around] is also in view also for some strange reason the search option brings weird results like songs that aren't even being searched for
